ui/cap-react/src/antd/admin/components/PropKeyEditorForm.js

```diff
@@ -23,6 +26,9 @@ const PropertyKeyEditorForm = ({
   } else {
     if (uiSchema["ui:widget"]) {
       type = uiSchema["ui:widget"];
+      if (uiSchema["ui:widget"] === "select") {
```

Comment:
We won't have those cases from the builder, only maybe for imported or manually created schemas, but I would say we should have those schemas follow the "form builder syntax" instead of implementing more custom checks that can make the code more confusing. What do you think?